addons/cargo/functions/fnc_unloadAllVehicles.sqf

```diff
@@ -0,0 +1,26 @@
+#include "script_component.hpp"
+/*
+ * Author: Vdauphin
+ * Unload all vehicles but not ACE Cargo.
+ *
+ * Arguments:
+ * 0: Vehicle <OBJECT>
+ *
+ * Return Value:
+ * Loaded vehiclesnot part of ACE Cargo <ARRAY>
+ *
+ * Example:
+ * [vehicle player] call ace_cargo_fnc_unloadAllVehicles
+ *
+ * Public: No
+ */
+
+params ["_vehicle"];
+
+private _loadedVehicles = getVehicleCargo _vehicle - (_vehicle getVariable [QGVAR(loaded), []]);
```

Comment:
`getVehicleCargo _vehicle - (_vehicle getVariable [QGVAR(loaded), []]);` is used in 2 places already, maybe move it to a function? `fnc_getVehicleCargo` to keep naming of the command that it's replacing basically.